app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please add a comment to say we're only doing this so we can connect to the server in the instrumentation tests?